src/menu/menu.scss

```diff
@@ -4,6 +4,98 @@
 @import '../inputs/variables';
 @import '../icon/index';
 
+@mixin iui-dropdown {
+  @include iui-reset;
+  @include iui-font-family;
+  width: 100%;
+  box-shadow: $iui-elevation-2;
```

Comment:
Do we really need to reuse stuff? Because the only reusable part is this `box-shadow` and `background-color` which are kind of already defined as a standard.

Reply:
Maybe it is ok to have that mixin to share that basic stuff... But I do agree with @bentleyvk here too, as its only 2 rules here.. 
If we start making mixins for every duplication of 2 rules, might end up in a huge mess :D

Reply:
I like the idea of a central location for all dropdown menu styling.  If we ever decided to change the styling, we could change it one place and have it propagate across all instances.  But I do understand the concern of introducing too many mixins.